onnxruntime/core/external_ops/pyop.h

```diff
@@ -0,0 +1,213 @@
+#pragma once
+#ifdef _WIN32
+#include <Windows.h>
+#define LIB_PYOP         "onnxruntime_pyop.dll"
+#define LOAD_PYOP_LIB(n) LoadLibraryA(n)
```

Comment:
Tried but failed, add comment to explain -  on linux, I need to pass "RTLD_GLOBAL" to dlopen, otherwise the initialization of Python C API will fail with "failed to import numpy library". 
Env() does allow for such flexibility.